apps/web/src/app/(dashboard)/admin/users/[id]/page.tsx

```diff
@@ -0,0 +1,126 @@
+'use client';
+
+import { useRouter } from 'next/navigation';
+
+import { zodResolver } from '@hookform/resolvers/zod';
+import { Loader } from 'lucide-react';
+import { Controller, useForm } from 'react-hook-form';
+
+import { trpc } from '@documenso/trpc/react';
+import { Button } from '@documenso/ui/primitives/button';
+import { Combobox } from '@documenso/ui/primitives/combobox';
+import { Input } from '@documenso/ui/primitives/input';
+import { Label } from '@documenso/ui/primitives/label';
+import { useToast } from '@documenso/ui/primitives/use-toast';
+
+import { FormErrorMessage } from '../../../../../components/form/form-error-message';
+import {
+  TUserFormSchema,
+  ZUserFormSchema,
+} from '../../../../../providers/admin-user-profile-update.types';
+
+export default function UserPage({ params }: { params: { id: number } }) {
+  const { toast } = useToast();
+  const router = useRouter();
+
+  const result = trpc.profile.getUser.useQuery(
+    {
+      id: Number(params.id),
+    },
+    {
+      enabled: !!params.id,
+    },
+  );
+
+  const user = result.data;
+
+  const roles = user?.roles;
+  let rolesArr: string[] = [];
+
+  if (roles) {
+    rolesArr = Object.values(roles);
+  }
```

Comment:
Isn't roles always going to be an array of enums for this? Theoretically we can just have it be 
`const roles = user?.roles ?? []`

Reply:
True, I just gave it a try. Thanks for the catch.